src/core/tracking.js

```diff
@@ -131,10 +136,7 @@ export function getAction(type) {
 }
 
 export default new Tracking({
-  trackingEnabled: (
-    convertBoolean(config.get('trackingEnabled')) &&
-    isDoNotTrackEnabled()
-  ),
+  trackingEnabled: convertBoolean(config.get('trackingEnabled')),
```

Comment:
I don't think this variable is useful. It's redundant and makes the code more complicated and harder to test (e.g. there is no test coverage of the config reading functionality at all). Let's *not* fix it in this patch though :) because it will require updating the tests. I'll file a lower priority follow-up.